packages/block-editor/src/hooks/layout-child.js

```diff
@@ -44,9 +59,20 @@ function useBlockPropsChildLayoutStyles( { style } ) {
 		 * columnCount is set, the grid is responsive so a
 		 * container query is needed for the span to resize.
 		 */
-		if ( columnSpan && ( minimumColumnWidth || ! columnCount ) ) {
-			// Calculate the container query value.
-			const columnSpanNumber = parseInt( columnSpan );
+		if (
+			( columnSpan || columnStart ) &&
+			( minimumColumnWidth || ! columnCount )
+		) {
+			// Check if columnSpan and columnStart are numbers so Math.max doesn't break.
+			const columnSpanNumber = columnSpan ? parseInt( columnSpan ) : null;
+			const columnStartNumber = columnStart
+				? parseInt( columnStart )
```

Comment:
Just a drive-by comment, do we need to check that `parseInt` was successful (i.e. an `isNaN` check of something similar to how `parentColumnValue` is checked), or can we always trust that if it was truthy, it'll be a number? Same with the line below.

Reply:
If parseInt isn't successful it returns NaN, which will evaluate to false. Any other result should be a number 😅

Reply:
Ah, gotcha. I noticed the value winds up being used directly in the calculation for `containerQueryValue` so wondered if there was a chance `NaN` could be unexpectedly output in the container query line? Not sure if that would ever be reached in practice, though, so feel free to ignore this comment for now!